Language/Variables/Data Types/String/Functions/c_str.adoc

```diff
@@ -19,10 +19,7 @@ Converte o conteúdo de um objeto String para uma string estilo C, terminada em
 
 [float]
 === Sintaxe
-[source,arduino]
-----
-string.c_str()
-----
+`meuString.c_str()`

```

Comment:
Hey per.
Portuguese is a very gender oriented language and "string" is considered feminine in Portuguese, so we use the feminine version of the possessive pronoun "minha" instead of "meu" which is used for masculine subjects.
If you can kindly rewrite the PR with "minha" instead of "meu", it would be great.
Thanks!

Reply:
Of course string would be feminine, but surely String must be masculine? I'm only joking! Thanks for catching my mistake. I have fixed it via a force push to keep this to a single commit (and hide my shame).

Reply:
Hahaha, no problem. And not a shame at all.